### ISSUE TITLE
lingering build/load error messages

### ISSUE THREAD

**Issue by turbomam** [microbiomedata/nmdc-schema#428]
closes #347

**Comment by turbomam** [comment]
```

INFO:root:class "database" slot "date created" does not reference an existing slot.  New slot was created.
INFO:root:class "database" slot "etl software version" does not reference an existing slot.  New slot was created.
INFO:root:class "database" slot "nmdc schema version" does not reference an existing slot.  New slot was created.
INFO:root:class "metaproteomics analysis activity" slot "has peptide quantifications" does not reference an existing slot.  New slot was created.
INFO:root:class "pathway" slot "has_part" does not reference an existing slot.  New slot was created.
INFO:root:class "reaction participant" slot "chemical" does not reference an existing slot.  New slot was created.
INFO:root:class "reaction participant" slot "stoichiometry" does not reference an existing slot.  New slot was created.
INFO:root:class "study" slot "notes" does not reference an existing slot.  New slot was created.

INFO:root:Default_range not specified. Default set to 'string'


WARNING:root:No namespace defined for URI: https://microbiomedata/schema/workflow_execution_activity/asm_score
WARNING:root:No namespace defined for URI: https://microbiomedata/schema/workflow_execution_activity/binned_contig_num
WARNING:root:No namespace defined for URI: https://microbiomedata/schema/workflow_execution_activity/contig_bp
WARNING:root:No namespace defined for URI: https://microbiomedata/schema/workflow_execution_activity/contigs
WARNING:root:No namespace defined for URI: https://microbiomedata/schema/workflow_execution_activity/ctg_L50
WARNING:root:No namespace defined for URI: https://microbiomedata/schema/workflow_execution_activity/ctg_L90
WARNING:root:No namespace defined for URI: https://microbiomedata/schema/workflow_execution_activity/ctg_logsum
WARNING:root:No namespace defined for URI: https://microbiomedata/schema/workflow_execution_activity/ctg_max
WARNING:root:No namespace defined for URI: https://microbiomedata/schema/workflow_execution_activity/ctg_N50
WARNING:root:No namespace defined for URI: https://microbiomedata/schema/workflow_execution_activity/ctg_N90
WARNING:root:No namespace defined for URI: https://microbiomedata/schema/workflow_execution_activity/ctg_powsum
WARNING:root:No namespace defined for URI: https://microbiomedata/schema/workflow_execution_activity/gap_pct
WARNING:root:No namespace defined for URI: https://microbiomedata/schema/workflow_execution_activity/gc_avg
WARNING:root:No namespace defined for URI: https://microbiomedata/schema/workflow_execution_activity/gc_std
WARNING:root:No namespace defined for URI: https://microbiomedata/schema/workflow_execution_activity/has_calibration
WARNING:root:No namespace defined for URI: https://microbiomedata/schema/workflow_execution_activity/has_metabolite_quantifications
WARNING:root:No namespace defined for URI: https://microbiomedata/schema/workflow_execution_activity/input_base_count
WARNING:root:No namespace defined for URI: https://microbiomedata/schema/workflow_execution_activity/input_contig_num
WARNING:root:No namespace defined for URI: https://microbiomedata/schema/workflow_execution_activity/input_read_bases
WARNING:root:No namespace defined for URI: https://microbiomedata/schema/workflow_execution_activity/input_read_count
WARNING:root:No namespace defined for URI: https://microbiomedata/schema/workflow_execution_activity/lowDepth_contig_num
WARNING:root:No namespace defined for URI: https://microbiomedata/schema/workflow_execution_activity/mags_list
WARNING:root:No namespace defined for URI: https://microbiomedata/schema/workflow_execution_activity/MAGsAnalysisActivity
WARNING:root:No namespace defined for URI: https://microbiomedata/schema/workflow_execution_activity/MetabolomicsAnalysisActivity
WARNING:root:No namespace defined for URI: https://microbiomedata/schema/workflow_execution_activity/metagenome_assembly_parameter
WARNING:root:No namespace defined for URI: https://microbiomedata/schema/workflow_execution_activity/MetagenomeAnnotationActivity
WARNING:root:No namespace defined for URI: https://microbiomedata/schema/workflow_execution_activity/MetagenomeAssembly
WARNING:root:No namespace defined for URI: https://microbiomedata/schema/workflow_execution_activity/MetaproteomicsAnalysisActivity
WARNING:root:No namespace defined for URI: https://microbiomedata/schema/workflow_execution_activity/MetatranscriptomeActivity
WARNING:root:No namespace defined for URI: https://microbiomedata/schema/workflow_execution_activity/MetatranscriptomeAnnotationActivity
WARNING:root:No namespace defined for URI: https://microbiomedata/schema/workflow_execution_activity/MetatranscriptomeAssembly
WARNING:root:No namespace defined for URI: https://microbiomedata/schema/workflow_execution_activity/NomAnalysisActivity
WARNING:root:No namespace defined for URI: https://microbiomedata/schema/workflow_execution_activity/num_aligned_reads
WARNING:root:No namespace defined for URI: https://microbiomedata/schema/workflow_execution_activity/num_input_reads
WARNING:root:No namespace defined for URI: https://microbiomedata/schema/workflow_execution_activity/output_base_count
WARNING:root:No namespace defined for URI: https://microbiomedata/schema/workflow_execution_activity/output_read_bases
WARNING:root:No namespace defined for URI: https://microbiomedata/schema/workflow_execution_activity/output_read_count
WARNING:root:No namespace defined for URI: https://microbiomedata/schema/workflow_execution_activity/read_QC_analysis_statistic
WARNING:root:No namespace defined for URI: https://microbiomedata/schema/workflow_execution_activity/ReadBasedTaxonomyAnalysisActivity
WARNING:root:No namespace defined for URI: https://microbiomedata/schema/workflow_execution_activity/ReadQCAnalysisActivity
WARNING:root:No namespace defined for URI: https://microbiomedata/schema/workflow_execution_activity/scaf_bp
WARNING:root:No namespace defined for URI: https://microbiomedata/schema/workflow_execution_activity/scaf_L50
WARNING:root:No namespace defined for URI: https://microbiomedata/schema/workflow_execution_activity/scaf_L90
WARNING:root:No namespace defined for URI: https://microbiomedata/schema/workflow_execution_activity/scaf_l_gt50K
WARNING:root:No namespace defined for URI: https://microbiomedata/schema/workflow_execution_activity/scaf_logsum
WARNING:root:No namespace defined for URI: https://microbiomedata/schema/workflow_execution_activity/scaf_max
WARNING:root:No namespace defined for URI: https://microbiomedata/schema/workflow_execution_activity/scaf_N50
WARNING:root:No namespace defined for URI: https://microbiomedata/schema/workflow_execution_activity/scaf_N90
WARNING:root:No namespace defined for URI: https://microbiomedata/schema/workflow_execution_activity/scaf_n_gt50K
WARNING:root:No namespace defined for URI: https://microbiomedata/schema/workflow_execution_activity/scaf_pct_gt50K
WARNING:root:No namespace defined for URI: https://microbiomedata/schema/workflow_execution_activity/scaf_powsum
WARNING:root:No namespace defined for URI: https://microbiomedata/schema/workflow_execution_activity/scaffolds
WARNING:root:No namespace defined for URI: https://microbiomedata/schema/workflow_execution_activity/too_short_contig_num
WARNING:root:No namespace defined for URI: https://microbiomedata/schema/workflow_execution_activity/unbinned_contig_num
WARNING:root:No namespace defined for URI: https://microbiomedata/schema/workflow_execution_activity/WorkflowExecutionActivity

WARNING:root:Unrecognized prefix: biolink
WARNING:root:Unrecognized prefix: COG
WARNING:root:Unrecognized prefix: EC
WARNING:root:Unrecognized prefix: gtpo
WARNING:root:Unrecognized prefix: MetaCyc
WARNING:root:Unrecognized prefix: MetaNetX
WARNING:root:Unrecognized prefix: RetroRules
WARNING:root:Unrecognized prefix: skos
WARNING:root:Unrecognized prefix: UniProtKB
```

**Comment by turbomam** [comment]
resolved, with the exception of #451
